android-runtime/runtime/src/main/java/io/sdkgen/runtime/SdkgenHttpClient.kt

```diff
@@ -119,12 +119,18 @@ open class SdkgenHttpClient(
     private val random = Random()
     private val hexArray = "0123456789abcdef".toCharArray()
     private val gson = Gson()
-    private val httpClient = OkHttpClient.Builder()
-        .connectTimeout(5, TimeUnit.MINUTES)
-        .readTimeout(5, TimeUnit.MINUTES)
-        .callTimeout(5, TimeUnit.MINUTES)
-        .writeTimeout(5, TimeUnit.MINUTES)
-        .build()
+    private val logging : HttpLoggingInterceptor = HttpLoggingInterceptor().apply {
+        this.level = HttpLoggingInterceptor.Level.BODY
+        this.redactHeader("Authorization")
+        this.redactHeader("Cookie")
+    }
```

Comment:
Logging is actually harder than this: sdkgen doesn't have any of those headers, you should redact properties with the annotation `!secret`

Reply:
Where can I see these properties?

Reply:
Probably from the AST, you may want to add something in kotlin-generator to make it easier for you

Reply:
This would require another aproach 🤔
Maybe create class wraping String and handle this type on HTTP login 🤔